File: tests/openmp_examples/program_control/error.1.f90
!$omp  metadirective when(implementation={vendor(gnu)}: nothing) otherwise(error at(compilation) severity(fatal) message("GNU compiler required."))
!$omp     error at(execution) severity(fatal) message("3 or more procs required.")
!$omp   parallel master
!$omp    error at(compilation) severity(warning) message("Notice: masteris deprecated.")
!$omp    error at(execution) severity(warning) message("Notice: masked to be used in next release.")
!$omp   end parallel master

File: tests/openmp_examples/program_control/error.1.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp metadirective when(implementation={vendor(gnu)}: nothing ) otherwise(error at(compilation) severity(fatal) message("GNU compiler required."))
#pragma omp error at(execution) severity(fatal) message("3 or more procs required.")
#pragma omp parallel master
#pragma omp error at(compilation) severity(warning) message("Notice: master is deprecated.")
#pragma omp error at(execution) severity(warning) message("Notice: masked used next release.")
